docs/reference/commands/cli-jvm-options.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[cli-tool-jvm-options]]
+=== JVM options for CLI tools
```

Comment:
Add a `=` here to make it a sub-section.

You'll want to include `{tool-name}` in the anchor above or we'll end up with duplicate anchors and those have to be unique across the whole project.